From 12135d950730171c7766eedcf31b572a935f428e Mon Sep 17 00:00:00 2001
From: Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
Date: Thu, 9 Jun 2016 12:45:14 +0200
Subject: [PATCH 4/6] inject snapped JRE path in build

---
 jvmfwk/source/framework.cxx | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/jvmfwk/source/framework.cxx
+++ b/jvmfwk/source/framework.cxx
@@ -66,8 +66,9 @@ javaFrameworkError jfw_findAllJREs(std::
         //get the list of paths to jre locations which have been
         //added manually
         const jfw::MergedSettings settings;
-        const std::vector<OUString>& vecJRELocations =
-            settings.getJRELocations();
+        std::vector<OUString> vecJRELocations = settings.getJRELocations();
+        OUString sSnappedJRE("file:///snap/libreoffice/current/usr/lib/jvm/java-11-openjdk-amd64/"); // FIXME: use SNAP environment variable
+        vecJRELocations.insert(vecJRELocations.begin(), { sSnappedJRE } );
         //Use every plug-in library to get Java installations.
         for (auto const & vendor: vecVendors)
         {
