diff --git a/Makefile.in b/Makefile.in
index 6af416b8a..b690e5c5a 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -59,12 +59,7 @@ else # MAKE_RESTARTS
 .DEFAULT_GOAL := build
 
 check-if-root:
-	@if test ! `uname` = 'Haiku' -a `id -u` = 0 && ! grep -q 'lxc\|docker' /proc/self/cgroup && ! grep -q 'libpod_parent' /proc/self/cgroup; then \
-		echo; \
-		echo 'Building LibreOffice as root is a very bad idea, use a regular user.'; \
-		echo; \
-		exit 1; \
-	fi
+	@true
 
 gb_Side ?= host
 
