Description: look up the HSQLDB jar inside the snap
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/connectivity/source/drivers/hsqldb/HDriver.cxx
+++ b/connectivity/source/drivers/hsqldb/HDriver.cxx
@@ -224,7 +224,7 @@ namespace connectivity
                 aProperties.put( "JavaDriverClassPath",
                     OUString(
 #ifdef SYSTEM_HSQLDB
-                        HSQLDB_JAR
+                        "vnd.sun.star.expand:file://$SNAP" HSQLDB_JAR
 #else
                         "vnd.sun.star.expand:$LO_JAVA_DIR/hsqldb.jar"
 #endif
--- a/RepositoryExternal.mk
+++ b/RepositoryExternal.mk
@@ -2887,7 +2887,7 @@ define gb_LinkTarget__use_hsqldb
 
 $(call gb_LinkTarget_add_defs,$(1),\
 	-DSYSTEM_HSQLDB \
-	-DHSQLDB_JAR=\""file://$(HSQLDB_JAR)"\" \
+	-DHSQLDB_JAR=\""$(HSQLDB_JAR)"\" \
 )
 
 endef
