diff --git a/external/libwpd/ExternalProject_libwpd.mk b/external/libwpd/ExternalProject_libwpd.mk
index feee6e0..72b2d4f 100644
--- a/external/libwpd/ExternalProject_libwpd.mk
+++ b/external/libwpd/ExternalProject_libwpd.mk
@@ -30,6 +30,7 @@ $(call gb_ExternalProject_get_state_target,libwpd,build) :
 			--without-docs \
 			--disable-tools \
 			--disable-debug \
+			--disable-werror \
 			$(if $(filter MACOSX,$(OS)), \
 				--disable-werror \
 				--prefix=/@.__________________________________________________OOO) \
