From 393d44c9cf9dc2c5f7790ed7775632ac58a54e7e Mon Sep 17 00:00:00 2001
From: Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
Date: Wed, 8 Jun 2016 18:42:42 +0200
Subject: [PATCH 2/6] dont ftbfs on warnings in libwpd

---
 external/libwpd/ExternalProject_libwpd.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/external/libwpd/ExternalProject_libwpd.mk b/external/libwpd/ExternalProject_libwpd.mk
index feee6e0..72b2d4f 100644
--- a/external/libwpd/ExternalProject_libwpd.mk
+++ b/external/libwpd/ExternalProject_libwpd.mk
@@ -30,6 +30,7 @@ $(call gb_ExternalProject_get_state_target,libwpd,build) :
 			--without-docs \
 			--disable-tools \
 			--disable-debug \
+			--disable-werror \
 			$(if $(filter MACOSX,$(OS)), \
 				--disable-werror \
 				--prefix=/@.__________________________________________________OOO) \
-- 
2.7.4

